src/collision/broad_phase.rs

```diff
@@ -206,11 +207,16 @@ type AabbIntervalQueryData = (
 #[allow(clippy::type_complexity)]
 fn add_new_aabb_intervals(
     added_aabbs: Query<AabbIntervalQueryData, (Added<ColliderAabb>, Without<ColliderDisabled>)>,
-    aabbs: Query<AabbIntervalQueryData>,
+    aabbs: Query<AabbIntervalQueryData, Without<ColliderDisabled>>,
     mut intervals: ResMut<AabbIntervals>,
     mut re_enabled_colliders: RemovedComponents<ColliderDisabled>,
+    mut re_enabled_entities: RemovedComponents<Disabled>,
 ) {
-    let re_enabled_aabbs = aabbs.iter_many(re_enabled_colliders.read());
+    let re_enabled_aabbs = aabbs.iter_many(
+        re_enabled_colliders
+            .read()
+            .chain(re_enabled_entities.read()),
+    );
```

Comment:
This is gross, but we'll handle it better with observers or hooks once we rework the broad phase to use a BVH.